### PR TITLE
CIV-7117 Recfor Listing for hearing Respondent Noft.

### PR DESCRIPTION
**Pull request by karthick-mohanasundaram-HMCTS**
**Before creating a pull request make sure that:**

- [x] commit messages are meaningful and follow good commit message guidelines
- [x] README and other documentation has been updated / added (if needed)
- [x] tests have been updated / new tests has been added (if needed)
- [x] Screenshot attached for the Evidence of manual testing
- [ ] Functional, Smoke and API tests have been run locally
- [x] Full gradle build and run tests with coverage have been completed (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/CIV-7117

### Screen Shot ###
Refer comment section in Jira ticket

### Change description ###
Recfor Listing for hearing Respondent Noft.

- Respondents should not receive notification for without notice application
- Respondent should receive notification for with notice application.

### Testing branch and instruction ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Comment by karthick-mohanasundaram-HMCTS** [comment]
image

**Comment by karthick-mohanasundaram-HMCTS** [comment]
image

**Review by pliao-hmcts — DISMISSED** [review]

**Comment by karthick-mohanasundaram-HMCTS** [comment]
image

**Comment by karthick-mohanasundaram-HMCTS** [comment]
image

**Comment by karthick-mohanasundaram-HMCTS** [comment]
image

**Review by deepthidoppalapudihmcts — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/reform/civil/utils/JudicialDecisionNotificationUtil.java [hunk 1]
```diff
@@ -117,6 +118,11 @@ private static boolean isApplicationForSequentialWrittenRep(CaseData caseData) {
             && isRespondentPresent;
     }
 
+    public static boolean isWithoutNotice(CaseData caseData) {
```
Comment (Azam-Hmcts) [inline src/main/java/uk/gov/hmcts/reform/civil/utils/JudicialDecisionNotificationUtil.java:121]: kindly check  if there is already existing method for isWithoutNotice.

**Review by Azam-Hmcts — APPROVED** [review]

**Review by deepthidoppalapudihmcts — APPROVED** [review]

**Review by deepthidoppalapudihmcts — APPROVED** [review]

**Review by pliao-hmcts — APPROVED** [review]

**Review by vasudevganesanhmcts — APPROVED** [review]

**Pull request merged**